README.rst

```diff
@@ -5,7 +5,7 @@ This document describes the source code for the `Eclipse Paho <http://eclipse.or
 
 This code provides a client class which enable applications to connect to an `MQTT <http://mqtt.org/>`_ broker to publish messages, and to subscribe to topics and receive published messages. It also provides some helper functions to make publishing one off messages to an MQTT server very straightforward.
 
-It supports Python 2.7.9+ or 3.6+.
+It supports Python 3.7+.
```

Comment:
OPTIONAL: Python EOLs in < 30 days so support for it should probably be dropped as well but that can be done in a separate pull request in the coming weeks.  https://devguide.python.org/versions

Reply:
> OPTIONAL: Python EOLs in < 30 days so support for it should probably be dropped as well but that can be done in a separate pull request in the coming weeks. https://devguide.python.org/versions

Agreed - I'm in the process of dropping 3.7 from my repos as well